src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc

```diff
@@ -110,7 +110,7 @@ void NaviStartStreamRequest::on_event(const event_engine::Event& event) {
       }
       if (hmi_apis::Common_Result::REJECTED == code) {
         LOG4CXX_INFO(logger_, "StartStream response REJECTED ");
-        SendRequest();
+        RetryStartSession();
```

Comment:
@Anatoliy-Leshin, why earlier SDL does send Request in case `REJECTED`, but now you think it unnecessary?